File: src/Lesson3/java/Ex3.java
package Lesson3.java;
import java.util.Scanner;

public class Ex3 {

    public static void main (String[] args){
        System.out.println("Enter number: ");
        Scanner scanner = new Scanner(System.in);
        int number = scanner.nextInt();
        while (number >= 10){
            int sumNumber = 0;
            while (number > 0){
                int digit = number % 10;
                sumNumber += digit;
                number = number / 10;
            }
            number = sumNumber;

        }
        System.out.println("Result " + number);

    }
}

File: src/Lesson3/forkotlin/Ex3.kt
package Lesson3.forkotlin

fun main() {

    print("Enter number: ")
    var number = readln().toInt()

    while (number >= 10) {

        var sumNumber = 0

        while (number > 0) {

            val digit = number % 10
            sumNumber += digit
            number /= 10
        }
        number = sumNumber
    }
    println("Result: $number")
}
